docs/configuration.md

```diff
@@ -36,6 +36,22 @@ $ redis-server --loadmodule ./redisgraph.so THREAD_COUNT 4
 
 ---
 
+## CACHE_SIZE
+
+The number of queries cached in RedisGraph. When a new query is needed to be store, and the cache is full, meaning the cache has reached the size of `CACHE_SIZE`, it will evict the least recently used (LRU) entry.
```

Comment:
Suggest rewording the start of this line:
`The max number of queries for RedisGraph to cache. When a new query is encountered and the cache is full,`